### PR TITLE
Slightly enhance README 

### PR DESCRIPTION
**Pull request by ThomDietrich**
Just some small things that came to my mind when checking your repo 👋

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
 Instructions:
 1) Boot Ubuntu 18.04 Desktop Live CD
 2) Select "Try Ubuntu"
 3) Open terminal (Ctrl+Alt+t)
-4) wget https://raw.github.com/ghfields/rpooler/master/rpooler.sh
-5) chmod +x rpooler.sh
-6) sudo su
-7) ./rpooler.sh
+4) `wget https://raw.github.com/ghfields/rpooler/master/rpooler.sh`
+5) `sudo bash rpooler.sh
```
Comment (ghfields) [inline README.md]: This line is missing a terminating back quote character .  When it is there, it disabled the scripts prompts.  Therefore, I recommend removing the backquote all together .

File: README.md [hunk 2]
```diff
@@ -1,21 +1,17 @@
...
 
 Instructions:
 1) Boot Ubuntu 18.04 Desktop Live CD
 2) Select "Try Ubuntu"
 3) Open terminal (Ctrl+Alt+t)
-4) wget https://raw.github.com/ghfields/rpooler/master/rpooler.sh
-5) chmod +x rpooler.sh
-6) sudo su
-7) ./rpooler.sh
+4) `wget https://raw.github.com/ghfields/rpooler/master/rpooler.sh`
```
Comment (ghfields) [inline README.md]: Back quote characters are not needed.

File: README.md (hunk 1 above)
Comment (ThomDietrich) [reply README.md]: Ugh, sorry. Corrected.

File: README.md (hunk 2 above)
Comment (ThomDietrich) [reply README.md]: To the contrary, it is common practice to set commands in this format. I'd recommend it.

Btw you are always able to edit files of a PR yourself.

Best!

Comment (ghfields) [reply README.md]: Ah got it.  You can also see my use of markdown is also lacking.  Originally thought the backquote was for the the command line literally.  It does look better this way.

Thanks again for your input.

**Pull request merged**